private/model/api/shape.go

```diff
@@ -691,7 +691,7 @@ type {{ $.ShapeName }} struct {
 
 	{{- if $.Exception }}
 		{{- $_ := $.API.AddSDKImport "private/protocol" }}
-		RespMetadata protocol.ResponseMetadata
+		RespMetadata protocol.ResponseMetadata` + "`json:\"-\" xml:\"-\"`" + `
```

Comment:
Does json tag with - as value already get ignored in the current SDK?

Reply:
Yes it does